hw3/src/routes/List.jsx

```diff
@@ -0,0 +1,55 @@
+import React from "react";
+import "../style.css";
+
+//Card containing country info
+const Card = (country) => {
+  //Generic not found response for if the API is missing any data
+  const not_found = "Data not found";
+  return(
+    <div class="card">
+      <h1 class="card-header"> {country.name} </h1>
+      <img src={country.flag_png} alt={country.flag_alt || "No alternative text provided"} />
+      <p><strong>Official Language(s): </strong>{country.official_languages.join(", ") || not_found}</p>
+      <p><strong>Population: </strong>{country.population.toLocaleString() || not_found}</p>
+      <p><strong>GDP (Billions): </strong>{country.gdp_billions || not_found}</p>
+    </div>
+  );
+}
+
+//All country cards
+const Countries = (countries) => {
```

Comment:
Change this to const Card = ({ country }) => {} here you are destructring country which the react will know but when using 

`const Card = ( country ) => {} `,

This syntax means that the function Card takes a single argument named country. However, in the context of React, this can be misleading:
**Single Parameter as Props:** This pattern suggests that country itself is the entire props object, not just a single property of props. If used without destructuring, you might have to access properties using country.props.someProperty if country is understood to be the props object.